app/views/Mails/acceptrefuse/sendProposalApproved.scala.txt

```diff
@@ -58,8 +28,42 @@ Questions?
 Please do not respond to this email but rather use the web interface:
 @ConferenceDescriptor.current().conferenceUrls.cfpURL@routes.CallForPaper.showCommentForProposal(proposal.id)
 
+Note that if have posted multiple proposals and do not have received any email for the others, it is because their are still in review.
```

Comment:
à confirmer :
"any email for the others," -> "any email for the others yet,"

Reply:
Done